webapp/app/assessment/components/review/review.js

```diff
@@ -144,19 +144,21 @@ const ReviewPanel = props => {
   const i18n = useI18n()
   const location = useLocation()
 
-  useEffect(() => {
-    closeCommentThread(countryIso)
-  }, [countryIso, location])
-
   const isActive = R.pipe(R.defaultTo({}), R.isEmpty, R.not)(props.openThread)
-  const target = R.isNil(props.openThread) ? null : (props.openThread.target).join(',')
+  const target = R.isNil(props.openThread) ? null : props.openThread.target.join(',')
```

Comment:
destruct props?

Reply:
i touched the minimum in this file. It must be refactored.
can we think about this later on?

Reply:
ok